proxy/neon_core_api/neon_layouts.py

```diff
@@ -160,6 +161,8 @@ def from_json(
         version = ""
         revision = ""
 
+        default_token_name = os.environ.get("DEFAULT_TOKEN_NAME", _EVMConfigConst.NeonTokenName.value)
```

Comment:
Please create the property with the token name in the `Config` object.
And pass `config` to this method as a parameter

Reply:
I originally had it inside `Config` but thought it would be simpler just to pull the environment there. But you're right, better to do it this way.